# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Debug.h
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/Debug.h
--- a/gfx/angle/checkout/src/libANGLE/Debug.h	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/Debug.h	Fri Nov 08 17:06:00 2019 +0000
@@ -27,7 +27,7 @@
   public:
     virtual ~LabeledObject() {}
     virtual void setLabel(const Context *context, const std::string &label) = 0;
-    virtual const std::string &getLabel() const     = 0;
+    virtual const std::string &getLabel() const                             = 0;
 };
 
 class Debug : angle::NonCopyable
@@ -52,12 +52,14 @@
                        GLenum type,
                        GLuint id,
                        GLenum severity,
-                       const std::string &message) const;
+                       const std::string &message,
+                       gl::LogSeverity logSeverity) const;
     void insertMessage(GLenum source,
                        GLenum type,
                        GLuint id,
                        GLenum severity,
-                       std::string &&message) const;
+                       std::string &&message,
+                       gl::LogSeverity logSeverity) const;
 
     void setMessageControl(GLenum source,
                            GLenum type,